ll-property-image.html

```diff
@@ -298,63 +320,49 @@
     },
 
     ready: function() {
-      this.imgTitle = this.imgTitle || this.imgName;
-      this.imgDescription = this.imgDescription || '';
+      this.title = this.title || this.name;
+      this.description = this.description || '';
       this.isDefaultImage = this.isDefaultImage || false;
```

Comment:
Seems to be fine. I think this came from when I was working through an issue with a watch not firing, however the root cause of that issue ended up being something different.